app/assets/stylesheets/custom/agents.scss

```diff
@@ -186,3 +186,10 @@
   opacity: 0;
 }
 
+.notransition {
```

Comment:
could be nicer the other way around: only apply transitions when has `.transition` class